### ISSUE TITLE
Randomize Playlist button does not work

### ISSUE THREAD

**Issue by takaji10** [TT-ReBORN/Georgia-ReBORN#16]
If I click on the Randomize Playlist button in the playbar, my playlist view scrolls to the top automatically and playback remains in the default state.

**Comment by TT-ReBORN** [comment]
What do you mean exactly? 
If you click on the randomize button, it randomizes the whole playlist.

**Comment by takaji10** [comment]
Hopefully I caught it clearly here: https://i.gyazo.com/50799a405afdddc5174008d4b146fc48.gif

**Comment by TT-ReBORN** [comment]
Try to unlock your playlist.

**Comment by takaji10** [comment]
Ahh, I understand the intended use now. I deselected "Force-sorted" on the playlist and now I can see that clicking on "Randomize Playlist" randomly-sorts the entire playlist.

I expected this button to change the playback order to random, and not randomly-sort the whole playlist. At 33k tracks, clicking that button introduces a fairly long lag while the playlist sorts and then loads.

I have a subsequent feature request, which I'll post as an idea in the discussions area, just to keep this clean here. Closing this as it's not a bug. :)
